client-src/elements/form-field-specs.js

```diff
@@ -482,6 +482,37 @@ export const ALL_FIELDS = {
     label: 'TAG Review',
     help_text: html`Link(s) to TAG review(s), or explanation why this is
                 not needed.`,
+    extra_help: html`
+    <p>
+      The <a target="_blank" href="https://www.w3.org/2001/tag/">W3C Technical Architecture Group</a> (TAG) 
+      is a special working group of the W3C that consists of a few appointed and elected members, 
+      all of whom are experienced members of the web standards community. 
+      The Blink launch process has a formal requirement for requesting a 
+      <a target="_blank" href="https://github.com/w3ctag/design-reviews">TAG review</a> for all features. 
+      The review happens publicly on a GitHub issue.
+    </p>
+    <p>
+      You will likely have asked for an "<a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=005-early-design-review.md"
+      >Early Design Review</a>" earlier in the process to get the TAG familiar with your feature. 
+      This isn't that.
+    </p>
+    <p>
+      It's recommended that you file a TAG <a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=010-specification-review.md"
+      >Specification Review</a> at the I2E stage (right after is also fine). 
```

Comment:
```suggestion
      It's recommended that you file a TAG <a target="_blank" href=
        "https://github.com/w3ctag/design-reviews/issues/new?template=010-specification-review.md"
      >Specification Review</a> as soon as your specification is written, during your Origin Trial. 
```

Reply:
I was going to ask what the I2E was.  Yet another name for Origin Trial, I guess.

Reply:
I2E is "Intent to Experiment", which is the email the feature owner sends to blink-dev to get permission to run an Origin Trial. https://www.chromium.org/blink/launching-features/#step-5-optional-origin-trial describes this bit of the process.

I think of the I2E as an event (you press 'send' in your mail client) while "preparing for I2E" and "running an Origin Trial" are stages, since they last for a period of time. That might not align with what Chrome Status calls a "stage".